index.bs

```diff
@@ -4892,6 +4892,14 @@ TODO: Move it as a hook in the html spec instead.
    1. If [=geolocation overrides map=] [=map/contains=] |user context|,
       [=set emulated position data=] with |navigable| and [=geolocation overrides map=][|user context|].
 
+   1. If [=forced colors mode theme overrides map=] [=map/contains=] |user context|:
+
+      1. Let |theme| be [=forced colors mode theme overrides map=][|user context|].
+
+      1. If |theme| is null, set |theme| to "<code>none</code>".
+
+      1. [=Set emulated forced colors theme data=] with |navigable| and |theme|.
```

Comment:
@alisonmaher, as far as I understood [this is the hook](https://drafts.csswg.org/css-color-adjust-1/#set-emulated-forced-colors-theme-data) which you're using here, could you link in the document header?

Reply:
@lutien that's correct, and the link seems to work if you go to my hosted version of the spec updates: https://alisonmaher.github.io/alisonmaher/webdriver-bidi/index.html#module-emulation-commands.

Do you mean link it in the PR description? Or is there some documentation for these sorts of external links I should add in the doc itself?

Reply:
> @lutien that's correct, and the link seems to work if you go to my hosted version of the spec updates: https://alisonmaher.github.io/alisonmaher/webdriver-bidi/index.html#module-emulation-commands.
> 
hmm... it also works in the PR preview, so I guess it's somehow found automatically.

I meant adding it in the document like e.g. for geolocation override: https://github.com/w3c/webdriver-bidi/blob/main/index.bs#L201. But I guess if it works anyway, it's probably fine 🙂